Tools/Scripts/webkitdirs.pm

```diff
@@ -984,9 +984,9 @@ sub determineXcodeSDKPlatformName {
         $xcodeSDKPlatformNameExplanation ||= "via argument, `--simulator`";
         $simulatorIdiom = 'iPhone';
     }
-    if (checkForArgumentAndRemoveFromARGV("--ios-simulator")) {
+    if (checkForArgumentAndRemoveFromARGV("--iphone-simulator")) {
```

Comment:
Looks like this needs to be:
```
if (checkForArgumentAndRemoveFromARGV("--iphone-simulator") || checkForArgumentAndRemoveFromARGV("--ios-simulator"))
```
at least for a little while, since we can't easily synchronize a builedbot restart with this change.